### DISCUSSION TITLE
Remove the duplication of Apache Commons Lang library

### DISCUSSION THREAD

**Discussion by FJBDev** [mytourbook/mytourbook#952] [General]
In MyTourbook, the [Apache Commons Lang](https://commons.apache.org/proper/commons-lang/) is duplicated **at 3 multiple places** ([net.tourbook.ext.jars](https://github.com/mytourbook/mytourbook/blob/816164349000c8888aa81e4fbb57ab159059079f/bundles/net.tourbook.ext.jars/commons-lang3-3.12.0.jar), [net.tourbook.ext.geoclipse](https://github.com/mytourbook/mytourbook/blob/816164349000c8888aa81e4fbb57ab159059079f/bundles/net.tourbook.ext.geoclipse/lib/commons-lang3-3.9.jar), and [net.tourbook.ext.velocity](https://github.com/mytourbook/mytourbook/blob/816164349000c8888aa81e4fbb57ab159059079f/bundles/net.tourbook.ext.velocity/commons-lang3-3.12.0.jar)).
In order to remove redundancy, confusion and extra disk space, it would be useful to use only a unique jar.

I would suggest to have it located with all the other jars in [net.tourbook.ext.jars](https://github.com/mytourbook/mytourbook/tree/main/bundles/net.tourbook.ext.jars)

**Comment by wolfgang-ch** [comment]
👍

**Comment by FJBDev** [comment]
Addressed in this [PR](https://github.com/mytourbook/mytourbook/pull/953)
